docs/notes_20170414.md

```diff
@@ -45,12 +45,14 @@
 
     - Therefore a thread needs to hold a lock for the duration it is working a log record.
     - This version 1.0
+
 * This is not ideal since threads will have to wait for other threads for logging a message
     - Ideally we would want the log operation to be non blocking.
-    - Solution: Every thread can create a log record simultaneously and this is pushed to
+    - Solution: Every thread can create a log record simultaneously and a LogRecord is pushed to
     a thread safe queue by multiple producers. These records are then consumed by a single
-    thread to write to a log.
+    thread to write to a LogFile.
```

Comment:
Planning to use [http://www.open-std.org/JTC1/SC22/WG21/docs/papers/2013/n3533.html](url)